addons/l10n_nl/migrations/12.0.3.0/post-migration.py

```diff
@@ -5,8 +5,4 @@
 
 @openupgrade.migrate()
 def migrate(env, version):
-    openupgrade.delete_records_safely_by_xml_id(
-        env, [
-            'l10n_nl.tag_nl_13',
-        ],
-    )
+    openupgrade.set_xml_ids_noupdate_value('l10n_nl', ('tag_nl_13',), True)
```

Comment:
```suggestion
    openupgrade.set_xml_ids_noupdate_value(env, 'l10n_nl', ['tag_nl_13'], True)
```

Reply:
Ah my bad. I thought I tested it, but apparently I did not.. But I changed it and tested it this time.